media-libs/libjpeg-turbo/libjpeg_turbo-1.5.3.recipe

```diff
@@ -0,0 +1,138 @@
+SUMMARY="MMX, SSE, and SSE2 SIMD accelerated JPEG library"
+DESCRIPTION="libjpeg-turbo is a JPEG image codec that uses SIMD instructions \
+(MMX, SSE2, AVX2, NEON, AltiVec) to accelerate baseline JPEG compression and \
+decompression on x86, x86-64, ARM, and PowerPC systems. On such systems, \
+libjpeg-turbo is generally 2-6x as fast as libjpeg, all else being equal. On \
+other types of systems, libjpeg-turbo can still outperform libjpeg by a \
+significant amount, by virtue of its highly-optimized Huffman coding routines.
+In many cases, the performance of libjpeg-turbo rivals that of proprietary \
+high-speed JPEG codecs.
+libjpeg-turbo implements both the traditional libjpeg API as well as the less \
+powerful but more straightforward TurboJPEG API. libjpeg-turbo also features \
+colorspace extensions that allow it to compress from/decompress to 32-bit and \
+big-endian pixel buffers (RGBX, XBGR, etc.), as well as a full-featured Java \
+interface.
+libjpeg-turbo was originally based on libjpeg/SIMD, an MMX-accelerated \
+derivative of libjpeg v6b developed by Miyasaka Masaru. The TigerVNC and \
+VirtualGL projects made numerous enhancements to the codec in 2009, and in \
+early 2010, libjpeg-turbo spun off into an independent project, with the goal \
+of making high-speed JPEG compression/decompression technology available to \
+a broader range of users and developers."
+HOMEPAGE="https://www.libjpeg-turbo.org/"
+COPYRIGHT="1991-2018 Thomas G. Lane, Guido Vollbeding"
+LICENSE="BSD (3-clause)
+	IJG"
```

Comment:
We have a similiar license around called [Jpeg], should we use the same name for this port?

[Jpeg]: https://github.com/haikuports/haikuports/blob/c9e40d638522692285e21d08ee91c8371d7abe87/media-libs/jpeg/licenses/JPEG#L101

Reply:
The source is also licensed under the [Zlib license](https://github.com/libjpeg-turbo/libjpeg-turbo/blob/1.5.3/LICENSE.md)